lib/transatl.g

```diff
@@ -169,9 +169,8 @@ BindGlobal("CheckSynonyms", function(arg)
   for p in pairs do 
     Print("Checking pair ", p, "\n");
     for n in allnames do
-      pos := PositionSublist(n, p[1]);
-      if pos <> fail then
-        n2 := SubstitutionSublist(n, p[1], p[2]);
+      n2 := ReplacedString(n, p[1], p[2]);
+      if not IsIdenticalObj(n) <> IsIdenticalObj(n2) then
```

Comment:
@frankluebeck Thanks - this has been discovered and fixed almost straight after merging in https://github.com/gap-system/gap/commit/48edc20aea9be01037e2a7ec8fa71efd756d91cd